docs/other/monitor-kubernetes-kuantifier.md

```diff
@@ -0,0 +1,191 @@
+title: Monitor Kubernetes Workloads with Kuantifier
+DateReviewed: 2024-08-16
+
+Monitoring Kubernetes Workloads with Kuantifier
+===============================================
+
+Workload jobs run via Kubernetes will not integrate with [Gratia accounting](./troubleshooting-gratia/) by default.
+To report contributions to OSG made via Kubernetes, the [Kuantifier](kuantifier-github) helm chart can be installed
+into your cluster.
+
+Before Starting
+---------------
+
+### Confirm access to a running Kubernetes cluster
+
+All subsequent instructions assume you have access to a running Kubernetes cluster, and can run [kubectl](kubectl)
+against that cluster.
+
+### Install the Helm command line tools 
+
+Kuantifier itself, and several of its prerequisites, are installed via [helm chart](https://helm.sh/). The helm
+command line tools are used to install helm charts against a running kubernetes cluster, and can be installed
+as follows:
+
+1. Download the latest [helm release](helm-release)
+1. Unpack the release blob (eg. `tar -zxvf helm-v3.0.0-linux-amd64.tar.gz`)
+1. Move the `helm` binary from the archive into a location along your `$PATH` (eg. `mv linux-amd64/helm ~/.local/bin`) 
+
+### Install Prometheus and kube-state-metrics in your Kubernetes cluster
+
+Kuantifier relies on [Prometheus](prometheus) with [kube-state-metrics](kube-state-metrics) to account for pod resource usage. 
+There are a number of ways to install both, such as via the [prometheus community helm charts](prometheus-community):
+
+1. Add the prometheus community helm repository to your local helm 
+
+       :::console
+       helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
+       helm repo update
+
+1. Install the kube-state-metrics and prometheus helm charts into your kubernetes cluster.
+
+       :::console
+       helm install kube-state-metrics prometheus-community/kube-state-metrics
+       helm install prometheus prometheus-community/prometheus
+
+       !!! note
+       These commands will install into the `default` namespace by default. To install into a 
+       non-default namespace such as `monitoring`, use the `-n monitoring` flag in helm install.
+
+1. Ensure that the namespace where your workload pods run is properly configured.
```

Comment:
I'm a little nervous about being prescriptive about installation method here: I don't really want to be on the hook for supporting Prometheus/kube-state-metrics installs directly. I think it's fine if we have light recs on installation methods like you do above, maybe elaborating that the `prometheus-community` provides Helm charts for both

Also, if we remove the first two list items, a single enumerated item looks odd so we can get rid of the enumeration.